articlequality/utilities/extract_labelings.py

```diff
@@ -108,6 +108,7 @@ def extract_labelings(dump, extractor=None, verbose=False):
     :Returns:
         An iterator of dicts containing:
 
+        * talk_page_id -- The id of the article's talk page
         * page_title -- The normalized title of the article
```

Comment:
Should this be removed? Or maybe replaced by the talk page name?

Reply:
Yeah.  I think that's a good point.  We should call it the "talk_page_title".